src/Microsoft.ML.Analyzer/Microsoft.ML.Analyzer.csproj

```diff
@@ -2,8 +2,25 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard1.3</TargetFramework>
+    <IncludeAnalyzerInPackage>Microsoft.ML</IncludeAnalyzerInPackage>
   </PropertyGroup>
 
+  <Target Name="PrepareAnalyzerPackageAssets"
```

Comment:
Thinking a bit more on this, we could integrate this into the existing `PreparePackageAssets` target. I think it would help eliminate a little bit of duplication, and ensure one usage of the `PackageAsset` item.

Basically just change

[https://github.com/dotnet/machinelearning/blob/eb264892e75219b1f191d3c35fcbc636186b8acf/src/Directory.Build.targets#L5-L21](https://github.com/dotnet/machinelearning/blob/eb264892e75219b1f191d3c35fcbc636186b8acf/src/Directory.Build.targets#L5-L21)

to be:

```XML
  <Target Name="PreparePackageAssets"
          AfterTargets="Build">

    <ItemGroup Condition="'$(IncludeInPackage)' != ''">
      <PackageAsset Include="$(TargetPath)"
                    RelativePath="$(IncludeInPackage)\lib\$(TargetFramework)" />
      <PackageAsset Include="@(DebugSymbolsProjectOutputGroupOutput)"
                    RelativePath="$(IncludeInPackage)\lib\$(TargetFramework)" />
      <PackageAsset Include="@(DocumentationProjectOutputGroupOutput)"
                    RelativePath="$(IncludeInPackage)\lib\$(TargetFramework)" />
    </ItemGroup>

    <ItemGroup Condition="'$(IncludeAnalyzerInPackage)' != ''">
      <PackageAsset Include="$(TargetPath)"
                    RelativePath="$(IncludeAnalyzerInPackage)\analyzers\dotnet\cs" />
      <PackageAsset Include="@(DebugSymbolsProjectOutputGroupOutput)"
                    RelativePath="$(IncludeAnalyzerInPackage)\analyzers\dotnet\cs" />
    </ItemGroup>

    <Copy SourceFiles="@(PackageAsset)"
          DestinationFolder="$(PackageAssetsPath)%(PackageAsset.RelativePath)" />

  </Target>
```

Then all you need to do in this file is set `<IncludeAnalyzerInPackage>Microsoft.ML</IncludeAnalyzerInPackage>` like you are above.

Reply:
Great, thanks @eerhardt -- now updated, and I've rechecked the nuget.

---
In reply to: [219938652](https://github.com/dotnet/machinelearning/pull/999#discussion_r219938652) [](ancestors = 219938652)